scanpipe/forms.py

```diff
@@ -80,7 +80,7 @@ def clean_input_urls(self):
         Fetch the `input_urls` and sets the `downloads` objects in the cleaned_data.
         A validation error is raised if at least one URL can't be fetched.
         """
-        input_urls_str = self.cleaned_data.get("input_urls", "")
+        input_urls_str = self.cleaned_data.get("input_urls", "").lower()
```

Comment:
We should not tamper with the user input.
While trying to fix the user errors, you may be breaking valid inputs.
Here, the path component of a URL may be case-sensitive, depending on the web server and file system used.

I would suggest that we never modify the user-provided content but instead raise the proper error including a suggestion when possible.

See the behavior of Python exceptions for example:
```
>>> var = 1
>>> print(vr)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'vr' is not defined. Did you mean: 'var'?
```

I think we should simply validate the schema of the input_urls and raise an error if not in the supported schema list. 
As a second improvement, we could check if the `lower()` value of the schema is in the list and improve the error with a suggestion.
Something along: "Docker:// schema is not supported. Did you mean: 'docker://'?"